examples/accessors_attr/lib.rs

```diff
@@ -0,0 +1,207 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![feature(min_specialization)]
+
+pub use accessors_attr::*;
+
+#[openbrush::contract]
+pub mod accessors_attr {
+    use openbrush::{
+        traits::{
+            Storage,
+        },
+    };
+
+    #[ink(storage)]
+    #[derive(Storage)]
+    pub struct Contract {
+        // fields for hater logic
+        #[storage_field]
+        hated_logic: DumbData,
+    }
+
+    #[openbrush::upgradeable_storage(STORAGE_KEY)]
+    #[openbrush::accessors(DumbDataAccessors)]
+    #[derive(Storage)]
+    #[derive(Debug)]
+    pub struct DumbData {
+        #[get]
+        #[set]
+        dumb_g_s: u32,
```

Comment:
honestly I got lost during review in these variable names 😃 let's call it read_write, read_only, write_only, wdyt? 😃